mapping/requirements_officejs.json

```diff
@@ -2760,6 +2765,11 @@
 							"name": "TextBinding.setDataAsync",
 							"apiVersion": "1.1",
 							"availability": "GA"
+						},
+						{
```

Comment:
Updating just supportedRequirementSets should be sufficient. I'm not sure if/how supportedMethods is used, but these all seem to be older/original APIs.